plaidml/edsl/tests/BUILD

```diff
@@ -23,14 +23,14 @@ plaidml_cc_test(
         "//plaidml:intel_gen": [":edsl_test-skip.intel_gen.txt"],
         "//conditions:default": [],
     }),
-    linkstatic = True,
     toolchains = ["//plaidml:settings"],
     visibility = ["//visibility:public"],
     deps = [
         "//plaidml:testenv",
         "//plaidml/edsl",
         "//plaidml/exec",
     ],
+    linkstatic = True,
```

Comment:
Please use buildifer to format all bazel build files

Reply:
This was done, using the team standard pre-commit.  Not sure why buildifier is inconsistent here.